captum/attr/_core/neuron/neuron_ablation.py

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/env python3
+import torch
+
+from ..._utils.attribution import NeuronAttribution, PerturbationAttribution
+from ..._utils.common import _verify_select_column
+from ..._utils.gradient import _forward_layer_eval
+
+from ..feature_ablation import FeatureAblation
+
+
+class NeuronAblation(NeuronAttribution, PerturbationAttribution):
```

Comment:
I think the name, `NeuronAblation `, might be a little misleading, since people might understand that it is performing neuron ablation in sense of neuron deletion. Like deleting some of the neurons and observing its impact on the prediction. Would `neuron_feature_ablation.py (NeuronFeatureAblation` be a better name ?

Reply:
Makes sense, sounds good, sure NeuronFeatureAblation would work!